code/src/pages/MovieBio.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from "react";
+import { useParams, Link, useHistory } from "react-router-dom";
+
+import { BackButton } from "components/js/BackButton";
+
+import "components/css/moviebio.css";
+
+export const MovieBio = () => {
+  const { id } = useParams();
+  const [bio, setBio] = useState({});
+  const API_KEY = "88b00ce6c0c72d59de65e4b1fcce3a85";
```

Comment:
maybe put the api key either above or below the use-thingies? it feels like the odd one out in that group :-)